common/web/keyman-version/build.sh

```diff
@@ -62,12 +62,10 @@ export default class KEYMAN_VERSION {
 }
   " > ./version.inc.ts
 
-  # Note: in a dependency build, we'll expect keyman-version to be built by tsc -b
-  if builder_is_dep_build; then
-    echo "[$THIS_SCRIPT_IDENTIFIER] skipping tsc -b; will be completed by $builder_dep_parent"
-  else
-    npm run build -- $builder_verbose
-  fi
+  npm run tsc -- -b $builder_verbose
+  # kmlmc (the lexical model compiler) relies on a Node-based import, but after some of the earlier
+  # ES-modularization work, our main output's an ES module.  Fortunately, esbuild can provide an easy stopgap.
```

Comment:
It was converted to esmodules in the feature-ldml branch (and keyman-version ended up with both esmodule and legacy versions)